### DISCUSSION TITLE
Is it possible install at same time `@mikro-orm/mysql` and `@mikro-orm/mongodb`?

### DISCUSSION THREAD

**Discussion by alexojegu** [mikro-orm/mikro-orm#2900] [General]
I have a monorepo project with two branches `mysql` and `mongo` at the first I am using `@mikro-orm/mysql` and at the second `@mikro-orm/mongodb`.

But when I change the branch, for example from `mysql` to `mongo` I get:
> error TS2307: Cannot find module `@mikro-orm/mongodb` or its corresponding type declarations.

And when I install `@mikro-orm/mongodb`, it uninstall `@mikro-orm/mysql` and vice versa.

```
$ npm -w @my/project install @mikro-orm/mongodb

added 17 packages, removed 10 packages, and audited 1055 packages in 28s
```

**Comment by B4nan** [comment]
Yes of course, you can install all the drivers you want. If you are switching git branches and have different dependencies in each, you need to run `npm install` after you switch to a branch.

**Reply by alexojegu** [reply]
Ok, so I understand that this is normal behavior.

Thank you and excuse me.
